app/models/search/work_query.rb

```diff
@@ -275,20 +275,29 @@ def facet_tags?
     options[:faceted]
   end
 
-  def facet_collections?
+  def collected?
     options[:collected]
   end
 
   def include_restricted?
     User.current_user.present? || options[:show_restricted]
   end
 
+  # Include unrevealed works only if we're on a collection page
+  # OR the collected works page of a user
   def include_unrevealed?
```

Comment:
From old search, for comparison:

https://github.com/otwcode/otwarchive/blob/0a980356c82bad64cc248e7b5501106163a27864/app/models/work_search.rb#L312-L322